src/cards/CafeCardStyles.js

```diff
@@ -18,73 +18,70 @@ export const Image = styled.img`
 export const CardBackgroundColor = styled.div`
     height: ${props => props.shorterBackground ? '490px' : '640px'};
     width: 100%;
+    background: #77A6A1;
+    box-shadow: 0 0 10px 0 #B3B9B8;
+    border-radius: 5px;
     position: absolute;
     top: 50px;
-    background: #77A6A1;
-    box-shadow: 2px 3px 3px #53736A;
 `
 
 export const CafeInfo = styled.div`
     padding: 0 25px;
-    position: absolute;
-    text-align: center;
     width: 100%;
+    text-align: center;
+    color: #F2F5F5;
+    position: absolute;
     top: 250px;
-    color: #E4E9F2;
 `
 
 export const CafeTitle = styled.h2`
     font-family: Monaco;
-    font-weight: 400;
     font-size: 16px;
     letter-spacing: 1.74px;
+    font-weight: 400;
 `
 
-export const CafeAddress = styled.p`
-    font-family: Tahoma, sans-serif;
-    font-size: 14px;
-    letter-spacing: 0.64px;
-`
+export const CafeAddress = styled.p``
 
-export const CafeRatingStyle  = styled.section`
+export const RatingSection  = styled.section`
```

Comment:
As you use the styling of the rating labels twice you may define a general styling for all cases and specify it for each case with distinct styling properties.